frontend/packages/kubevirt-plugin/src/k8s/requests/vm/index.ts

```diff
@@ -0,0 +1,48 @@
+import { k8sPatch, resourceURL } from '@console/internal/module/k8s';
+import { getName, getNamespace } from '@console/shared';
+import { coFetch } from '@console/internal/co-fetch';
+import { analyticsSvc } from '@console/internal/module/analytics';
+import { getPxeBootPatch } from '../../patches/vm/vm-boot-patches';
+import { VirtualMachineModel } from '../../../models';
+import { VMKind } from '../../../types/vm';
+
+export enum VMActionType {
+  Start = 'start',
```

Comment:
should be uppercase `START`/`STOP`/`RESTART` as rest of the enums in console

Reply:
Some enums do use UPPER_CASE convention, but it seems that most of the enums use Capitalized one.